packages/components/src/components/modal/modal.tsx

```diff
@@ -158,6 +158,9 @@ export class Modal {
             ...parseAndGetAriaAttributes(this.aria),
           })}
           tabIndex={-1}
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          /* @ts-ignore */
+          inert={this.open ? null : true} // prevents focusable elements within nested open accordion
```

Comment:
Wouldn't it be better to set the attribute on the host element to be absolutely sure everything inside the component can't be focused?

Reply:
Probably makes no difference but adds the chance that the attribute can be removed.